docs/tutorials/customizing_robots.md

```diff
@@ -0,0 +1,216 @@
+
+## Customizing Action Spaces
+
+A robot is equipped with multiple controllers, each of which control a subset of the robot's low-level joint motors. Together, these controllers' inputs form the robot's corresponding action space. For example, a [Fetch](../reference/robots/fetch.html) robot consists of (a) a base controller controlling its two wheels, (b) a head controller controlling its two head joints, (c) an arm controller controlling its seven arm joints, and (d) a gripper controller controlling its two gripper joints (resulting in 13 DOF being controlled). An example set of controllers would be using a [DifferentialDriveController](../reference/controllers/dd_controller.html) for the base, [JointController](../reference/controllers/joint_controller.html)s for the head and arm, and binary  [MultiFingerGripperController](../reference/controllers/multi_finger_gripper_controller.html) for the gripper. In this case, the action space size would be 2 + 2 + 7 + 1 = 12. If we were to use an [InverseKinematicsController](../reference/controllers/ik_controller.html) commanding the 6DOF end-effector pose instead of the JointController for the arm, the action space size would be 2 + 2 + 6 + 1 = 11. Each of these controllers can be individual configured and swapped out for each robot.
+
+### Modifying Via Config
+
+One way to customize a robot's set of controllers is to manually set the desired controller configuration in the environment config file when creating an **OmniGibson** environment. An example is shown below:
+
+??? code "fetch_controller_cfg.yaml"
+    ``` yaml linenums="1"
+    robots:
+      - type: Fetch
+        position: [0, 0, 0]
+        orientation: [0, 0, 0, 1]
+        controller_config:
+          base:
+            name: DifferentialDriveController
+          arm_0:
+            name: InverseKinematicsController
+            kv: 2.0
+          gripper_0:
+            name: MultiFingerGripperController
+            mode: binary
+          camera:
+            name: JointController
+            use_delta_commands: False
+    ```
+
+In the above example, the types of controllers are specified for each component of the robot (`base`, `arm_0`, `gripper_0`, `camera`), and additional relevant keyword arguments to pass to the specific controller init calls can also be specified. If a controller or any keyword arguments are not specified for a given component, a default set of values will be used, which are specified in the robot class itself (`_default_controller_config` property). Please see the [Controllers](../modules/controllers.md) section for additional details on controller arguments.
```

Comment:
Should we also mention `action_normalize` somewhere here? I remembered it will somehow overwrite command_input_limits/command_output_limits

Reply:
good point! done